src/lib/components/workflow/workflow-counts.svelte

```diff
@@ -10,22 +13,48 @@
   } from '$lib/stores/workflows';
   import type { WorkflowStatus } from '$lib/types/workflows';
   import { decodePayload } from '$lib/utilities/decode-payload';
+  import { getExponentialBackoffSeconds } from '$lib/utilities/refresh-rate';
 
-  import WorkflowCountAll from './workflow-count-all.svelte';
   import WorkflowCountStatus from './workflow-count-status.svelte';
 
   $: namespace = $page.params.namespace;
   $: query = $page.url.searchParams.get('query');
 
   let statusGroups: { status: WorkflowStatus; count: number }[] = [];
+  let newStatusGroups: { status: WorkflowStatus; count: number }[] = [];
+  let refreshInterval: ReturnType<typeof setInterval>;
 
-  const fetchCounts = async () => {
-    const { count, groups } = await fetchWorkflowCountByExecutionStatus({
-      namespace,
-      query,
-    });
-    $workflowCount.totalCount = parseInt(count);
-    statusGroups = groups.map((group) => {
+  let attempt = 1;
+  const initialIntervalSeconds = 5;
+  const maxAttempts = 100;
+
+  onDestroy(() => {
+    clearNewCounts();
+  });
+
+  const setBackoffInterval = () => {
+    attempt += 1;
+    clearInterval(refreshInterval);
+    if (attempt <= 100) {
```

Comment:
Should this be `maxAttempts` ? Or is `100` just an arbitrary choice here

Reply:
Good catch, fixed.